### PR TITLE
feature(component): feature panel

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: [#200](https://app.zenhub.com/workspaces/ffam-prototype-5c615588801d5a103031639c/issues/unimelb/ffam-prototype/200)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 40ab4e3b4a7eda78e3db274cc688de8f610d6c75

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5ca42fc9a6376c00082a1e3a

File: components/cards/CardFeaturesPanel.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <div
+    :href="href"
+    class="card card--features-panel card--features-panel--head bg-white">
+    <div class="card__thumb">
+      <div
+        :style="`background-image: url('${thumb}')`"
+        class="card__thumb-img"/>
+    </div>
+    <div class="card__inner">
+      <div class="card__subheader">
```
Comment (TimMansell) [inline components/cards/CardFeaturesPanel.vue]: Don't need the `card_subheader` wrapping element

File: components/cards/_card-features-panel.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    text-transform: none;
+  }
+
+  .card__footer {
+    display: flex;
+    justify-content: center;
+    padding: 0 var(--vr) var(--vr) var(--vr);
+  }
+
+  .card__subheader {
```
Comment (TimMansell) [inline components/cards/_card-features-panel.css]: Remove this.  Design seems to match without this extra padding

File: components/cards/_card-features-panel.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    font-size: 24px;
+    font-weight: var(--fw-bold);
+    text-align: center;
+    text-transform: none;
+  }
+
+  .card__footer {
+    display: flex;
+    justify-content: center;
+    padding: 0 var(--vr) var(--vr) var(--vr);
```
Comment (TimMansell) [inline components/cards/_card-features-panel.css]: If you're just overriding the top padding then use `padding-top: 0` instead.

File: components/cards/CardFeaturesPanel.vue [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        <h5 class="card__title">{{ title }}</h5>
+        <p class="card__meta">{{ text }}</p>
+      </div>
+    </div>
+    <div class="card__footer">
+      <ButtonIcon
```
Comment (TimMansell) [inline components/cards/CardFeaturesPanel.vue]: move `v-if="href"` to `<div class="card__footer">` so the extra padding disappears.

File: components/cards/_card-features-panel.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+  .card__subheader {
+    margin: var(--vr) 0;
+  }
+
+  .card__meta {
```
Comment (TimMansell) [inline components/cards/_card-features-panel.css]: Don't need this by adding `margin-bottom: var(--vr );` to .card__title

File: components/cards/_card-features-panel.css [hunk 6]
```diff
@@ -0,0 +1,57 @@
+.card--features-panel {
+  .card__title {
```
Comment (TimMansell) [inline components/cards/_card-features-panel.css]: add margin-bottom: var(--vr );

**Pull request merged**